code/game/objects/effects/anomalies/anomalies_pyroclastic.dm

```diff
@@ -43,3 +44,32 @@
 	pyro.mind.special_role = ROLE_PYROCLASTIC_SLIME
 	pyro.mind.add_antag_datum(/datum/antagonist/pyro_slime)
 	pyro.log_message("was made into a slime by pyroclastic anomaly", LOG_GAME)
+
+///Bigger, meaner, immortal pyro anomaly
+/obj/effect/anomaly/pyro/big
+	immortal = TRUE
+	aSignal = null
+	releasedelay = 2
+	move_force = MOVE_FORCE_OVERPOWERING
+
+/obj/effect/anomaly/pyro/big/Initialize(mapload, new_lifespan, drops_core)
+	. = ..()
+
+	transform *= 3
+
+/obj/effect/anomaly/pyro/big/Bumped(atom/movable/bumpee)
+	. = ..()
+
+	if(isliving(bumpee))
+		var/mob/living/living = bumpee
+		living.dust()
+
+/obj/effect/anomaly/pyro/big/anomalyEffect(delta_time)
+	. = ..()
+
+	if(!.)
+		return
+
+	var/turf/turf = get_turf(src)
+	if(!isspaceturf(turf))
```

Comment:
What about `openspaceturf`? No chance of it actually encountering one where it is, but if an admin spawns one on tram will it overwrite the tram trench with lava?